prometheus/src/test/scala/filodb/prometheus/parse/ParserSpec.scala

```diff
@@ -465,6 +465,11 @@ class ParserSpec extends AnyFunSpec with Matchers {
     parseSuccessfully("timestamp(some_metric)")
     parseError("timestamp(some_metric[5m])") // reason : Expected instant vector, got range vector
     parseError("timestamp(some_metric, hello)") // reason : Expected only 1 arg, got 2
+
+    // Trailing Commas
+    parseSuccessfully("sum without(and, by, avg, count, alert, annotations,)(some_metric)")
+    parseSuccessfully("sum without(and, by, avg, count, alert, annotations, )(some_metric)")
+    parseSuccessfully("sum by(and, by, avg, count, alert, annotations, )(some_metric)")
```

Comment:
Could you add a couple for binary joins (i.e. `+`/`-`/`and`, etc), as well? The analogous clauses are `on` and `ignoring`, and the label lists used for each are called "`labelNameList`" in the grammar (I am not sure offhand why we allow more complex expressions in an aggregation's clause).

Reply:
Ah I see I was confused -- `labelNameList` is the one you already updated, and `parameterList` has nothing to do with a `by`/`without` clause:
```
aggregation
    : AGGREGATION_OP parameterList
    | AGGREGATION_OP (by | without) parameterList
    | AGGREGATION_OP parameterList ( by | without)
    ;
by:      BY labelNameList;
without: WITHOUT labelNameList;
```

Cool, I thought we needed more updates to the grammar but that's not the case.